avm/res/batch/batch-account/README.md

```diff
@@ -1205,4 +1205,4 @@ This section gives you an overview of all local-referenced module files (i.e., o
 
 | Reference | Type |
 | :-- | :-- |
```

Comment:
As per @ahmadabdalla's comment: _putting an example of the ipconfig block to test it out as it can be real use case_